packages/xsnap/src/ava-xs.js

```diff
@@ -9,10 +9,12 @@ const { promises } = require('fs');
 const path = require('path');
 const glob = require('glob');
 
-esmRequire('@agoric/install-ses');
-const bundleSource = esmRequire('@agoric/bundle-source').default;
+// eslint-disable-next-line no-global-assign
+require = esmRequire; // help vs-code navigate
```

Comment:
I suggest using:

```js
// eslint-disable-next-line no-global-assign
require = require('esm')(module);
```

and move this up to the first line of the file instead of defining `esmRequire` at all.

I only used `esmRequire` to avoid needing to disable the eslint warning.  It is a complete replacement for `require` and the old `require` doesn't need to be used for anything except bootstrapping the new `require`.